_posts/2017/2017-08-17-tracing-the-suspect.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Tracing the suspect - a microservices murder mystery"
+date: "2017-08-17 09:30:00"
+image: blog-header/tracing-the-suspect.jpg
```

Comment:
What about one of those?

https://pixabay.com/en/snow-lane-traces-snow-zig-zag-81156/
https://www.pexels.com/photo/cold-dark-eerie-fear-207985/
https://pixabay.com/en/sand-footsteps-footprints-beach-768783/